src/ansys/fluent/core/session.py

```diff
@@ -122,13 +124,22 @@ def id(self) -> str:
         """Return the session id."""
         return self.fluent_connection.id
 
-    def start_transcript(self) -> None:
+    def start_transcript(self, file_path: str = None) -> None:
         """Start streaming of Fluent transcript."""
```

Comment:
@prmukherj please can you document the new argument, which will be useful in the documentation

Reply:
Sure, will do it